elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -16,14 +16,13 @@ $nav-item-variables: (
     MaxWidth: 100%
   ),
   tray: (
-    BackgroundColor:  #{pfe-color(surface--lightest)},
-    Color:            #{pfe-color(text)}
+    BackgroundColor:  #{pfe-var(surface--lightest)},
+    Color:            #{pfe-var(text)}
   )
-);
-$variables: map-merge($variables, $nav-item-variables);
+));
 
 :host {
-  @include pfe-print-variables($variables);
+  // @include pfe-print-local($LOCAL-VARIABLES);
```

Comment:
should this be deleted

Reply:
Yes, good catch.  Navigation made me super nervous so I was hesitant to delete anything.